document-store/src/main/java/org/hypertrace/core/documentstore/postgres/PostgresCollection.java

```diff
@@ -189,6 +189,12 @@ public boolean updateSubDoc(Key key, String subDocPath, Document subDocument) {
     return false;
   }
 
+  @Override
+  public boolean updateSubDoc(Set<Key> keys, String subDocPath, Document subDocument) {
+    // empty
+    return true;
```

Comment:
Can you please add implementation for Postgres? I think this should be straightforward. You might just have to change the SQL query to use an `IN` clause, rather than `=`